.cicd/pipeline.yml

```diff
@@ -69,6 +69,30 @@ steps:
       - "queue=mac-anka-large-node-fleet"
     skip: $SKIP_MOJAVE
 
+  - label: ":darwin: macOS 10.15 - Build"
+    command:
+      - "brew install git automake libtool wget cmake gmp gettext doxygen graphviz lcov python@3"
+      - "git clone $BUILDKITE_REPO eosio.cdt"
+      - "cd eosio.cdt && if [[ $BUILDKITE_BRANCH =~ ^pull/[0-9]+/head: ]]; then git fetch -v --prune origin refs/pull/$(echo $BUILDKITE_BRANCH | cut -d/ -f2)/head; fi"
+      - "cd eosio.cdt && git checkout -f $BUILDKITE_COMMIT && git submodule update --init --recursive"
+      - "cd eosio.cdt && ./.cicd/build.sh"
+      - "cd eosio.cdt && tar -pczf build.tar.gz build && buildkite-agent artifact upload build.tar.gz"
+    plugins:
+      - chef/anka#v0.5.1:
+          no-volume: true
+          inherit-environment-vars: true
+          vm-name: 10.15.0_6C_14G_40G
+          vm-registry-tag: "clean::cicd::git-ssh::nas::brew::buildkite-agent"
+          modify-cpu: 12
+          modify-ram: 24
+          always-pull: true
+          debug: true
+          wait-network: true
+    agents:
+      - "queue=mac-anka-large-node-fleet"
+    skip: $SKIP_MOJAVE
```

Comment:
Should these be `SKIP_CATALINA`? Or maybe `SKIP_MAC` if the intention is actually to skip all mac builds.

Reply:
Good catch. In fact, we need to change it completely to match what EOSIO uses. I've made the updates.